src/PKSim.BatchTool/PKSim.BatchTool.csproj

```diff
@@ -2,6 +2,7 @@
 
   <PropertyGroup>
     <TargetFramework>net472</TargetFramework>
+    <RuntimeIdentifiers>win-x64;win7-x64</RuntimeIdentifiers>
```

Comment:
`win7-x64`?

Reply:
Yeah same question. What's this change abou t?

Reply:
Lol. To get it built locally on my comp (which is Win11 however).
To be honest, I don't even understand why this is required in the 1st place for msbuild.

Reply:
But without it, the build fails (s. previous failed action runs).